packages/desktop-client/src/components/autocomplete/Autocomplete.tsx

```diff
@@ -559,17 +561,21 @@ function SingleAutocomplete<T extends Item>({
                 )}
               </View>
             ) : (
-              <Tooltip
-                position="bottom-stretch"
+              <Popover
+                triggerRef={triggerRef}
+                placement="bottom start"
                 offset={2}
+                isOpen={isOpen}
+                onOpenChange={close}
+                isNonModal
                 style={{
-                  padding: 0,
+                  ...styles.darkScrollbar,
```

Comment:
Instead of putting this in a `style` prop, can we try using glamor to pass this to `className`? Maybe the inline `style` prop doesn't support pseudo selectors.

Reply:
Using glamor is a good idea! Updated it now, let me know how it looks!

Reply:
It still is not working, we need to check that the darkScrollbar is set in the children's container element.